src/Features/CSharp/Portable/CodeLens/DisplayInfoCSharpServices.cs

```diff
@@ -0,0 +1,348 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.CodeLens;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.CSharp.Diagnostics
+{
+    [ExportLanguageService(typeof(IDisplayInfoLanguageServices), LanguageNames.CSharp), Shared]
+    internal sealed class DisplayInfoCSharpServices : IDisplayInfoLanguageServices
```

Comment:
our pattern is for CSharp to be in the front of hte name.  So this would be CSharpDisplayInfoLanguageServices.